priv/perf/apps/load_test/test/load_tests/runner/deposits_test.exs

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Runner.DepositsTest do
+  use ExUnit.Case
+
+  @moduletag :deposits
+
+  alias ExPlasma.Encoding
+
+  test "deposits test" do
```

Comment:
this is just a happy path test for deposits test the can be run using `mix run` with LoadTest.TestRunner.run